Expecto/Expecto.fs

```diff
@@ -861,21 +861,27 @@ module Tests =
     |> Test.toTestCodeList
     |> Seq.iter (fun t -> printfn "%s" t.name)
 
-  /// Runs tests with supplied options. Returns 0 if all tests passed, =
-  /// otherwise 1
+  /// Runs tests with the supplied options. Returns 0 if all tests passed, otherwise 1
   let runTests config tests =
     let run = if config.parallel then runParallel else run
     Global.initialiseIfDefault
       { Global.defaultConfig with
           getLogger = fun name -> LiterateConsoleTarget(name, config.verbosity) :> Logger }
     run config.printer tests
 
-  /// Runs tests in this assembly with supplied command-line options. Returns 0 if all tests passed, otherwise 1
-  let runTestsInAssembly config args =
-    let tests =
-      match testFromAssembly (Assembly.GetEntryAssembly()) with
-      | Some t -> t
-      | None -> TestList ([], Normal)
+  /// Runs all given tests with the supplied command-line options. Returns 0 if all tests passed, otherwise 1
+  let runTestsWithArgs config args tests =
     let config, isList = args |> ExpectoConfig.fillFromArgs config
     let tests = tests |> config.filter
```

Comment:
When you're here changing the code, please make single-param functions be called without `|>`